game/gui/hud.py

```diff
@@ -1,11 +1,11 @@
 from typing import (
     cast,
 )
-
 import arcade
 from arcade import gui
 
 from engine import scripts
+from engine.core import SCREEN_WIDTH, SCREEN_HEIGHT
```

Comment:
Some style adjustments here. We're using the [Google Python style guide](https://google.github.io/styleguide/pyguide.html) for this project. I'm not too picky about following it to the letter, but when people submit changes that necessarily move code away from the style guide, I ask them to put it back.

There's two things to change here:
1: Import packages, not variables/constants/classes/etc. ([link](https://google.github.io/styleguide/pyguide.html#22-imports))
```python
from engine import (
  core,
  scripts,
)
```
Don't import `SCREEN_WIDTH` and `SCREEN_HEIGHT` directly.

The next thing violates [PEP8](https://peps.python.org/pep-0008/#imports) which is a broadly used standard for formatting Python code. I don't think I've ever worked in a place that didn't use PEP8.
The violation here was removing line 4: standard library imports have a blank line between them and third-party imports. So there should be a blank line between the `typing` import (standard library) and the `arcade` import (a third-party import).